docs/intro.rst

```diff
@@ -402,6 +404,8 @@ This is the current list of error and warning codes:
 +------------+----------------------------------------------------------------------+
 | W503 (*)   | line break occurred before a binary operator                         |
 +------------+----------------------------------------------------------------------+
+| W504 (\*^) | doc line too long (82 > 79 characters)                               |
```

Comment:
I'm pretty sure you don't need the `\` .. Edit: though maybe I'm wrong looking up above..

Reply:
I'm wondering if this should be an `E503` instead of `W504`, since this isn't really do do with "line breaks".

Reply:
Should it be an error rather than a warning though? I was trying to be a bit gentler to existing code. We could rename the heading of the W5 section to just say "Line Length" to match the E5 section